app/src/main/java/com/onboarding/parkingsystemjava/fragment/ConfigureParkingLotsDialog.java

```diff
@@ -0,0 +1,62 @@
+package com.onboarding.parkingsystemjava.fragment;
+
+import android.app.DialogFragment;
+import android.content.Context;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.annotation.Nullable;
+import com.onboarding.parkingsystemjava.databinding.ConfigureParkingLotsDialogFragmentMainActivityBinding;
+import com.onboarding.parkingsystemjava.listener.OnInputListener;
+import com.onboarding.parkingsystemjava.mvp.contract.SetParkingContract;
+import com.onboarding.parkingsystemjava.mvp.presenter.ConfigureParkingPresenter;
+import com.onboarding.parkingsystemjava.mvp.view.ConfigureParkingView;
+import com.onboarding.parkingsystemjava.utils.ConstantUtils;
+
+public class ConfigureParkingLotsDialog extends DialogFragment {
+
+    private SetParkingContract.SetParkingPresenter presenter;
+    private ConfigureParkingLotsDialogFragmentMainActivityBinding binding;
+    private OnInputListener inputListener;
+
+    public static ConfigureParkingLotsDialog newInstance() {
+        return new ConfigureParkingLotsDialog();
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        this.presenter = new ConfigureParkingPresenter(this.inputListener, new ConfigureParkingView(this, binding));
```

Comment:
why this is in onStart and no in onCreate or onCreateView?

Reply:
should I move it?

Reply:
moved to onCreateView()